packages/sanes-chrome-extension/src/routes/recovery-phrase/index.dom.spec.ts

```diff
@@ -1,23 +1,32 @@
 import { Store } from 'redux';
 import { RootState } from '../../store/reducers';
-import { getMnemonic } from '../signup/components/ShowPhraseForm';
 import { aNewStore } from '../../store';
 import TestUtils from 'react-dom/test-utils';
 import { whenOnNavigatedToRoute } from '../../utils/test/navigation';
 import { WELCOME_ROUTE } from '../paths';
 import { travelToRecoveryPhrase } from './test/travelToRecoveryPhrase';
-import { sleep } from '../../utils/timer';
+
 import { withChainsDescribe } from '../../utils/test/testExecutor';
+import { PersonaManager } from '../../logic/persona';
 
 withChainsDescribe('DOM > Feature > Recovery Phrase', () => {
   let store: Store<RootState>;
 
+  beforeAll(async () => {
+    // All tests in this file should use the same persona instance to improve test speed
```

Comment:
Remove this comment. We are doing this here wrongly:
- no persona needed & used in this test (for now)
- broken the feature rule of creating a persona and showing its recovery phrase using react, meaning we have not tested in a pure and real correct way the feature (which at the ends is what dom test do)

Because:
- We gain more making visual and clear the necessity of destroying personas due to the bugs we have had in the past (hangs builds due to open connections, mostly).

Reply:
If we remove the creation of the Persona in this test, then we need to remove this check:

```
    const phraseParagraph = paragraph.innerHTML;
    expect(phraseParagraph).toBe(PersonaManager.get().mnemonic);
```

But maybe the test can inject a testing mnemonic instead. I adapt it now.